pkg/operator/controllers/allocation_api_server.go

```diff
@@ -316,10 +318,13 @@ func (s *AllocationApiServer) handleAllocationRequest(w http.ResponseWriter, r *
 		if err != nil {
 			if apierrors.IsConflict(err) {
 				s.logger.Info("conflict error patching game server", "error", err, "sessionID", args.SessionID, "buildID", args.BuildID, "retry", i)
+				Allocations500ErrorsCounter.WithLabelValues(gs2.Labels[LabelBuildName]).Inc()
```

Comment:
Can we make this a 409 (conflict) error? Since we already know the reason the allocation failed :)

Reply:
@ghov just pushed up the changes to address this. We've added a new metric for 409 conflict errors.